src/eos/chemTab.hpp

```diff
@@ -47,13 +46,16 @@ class ChemTab : public ChemistryModel, public std::enable_shared_from_this<ChemT
     /**
      * Private function to compute predictedSourceEnergy, progressVariableSource, and massFractions
      * @param density, the density is used to scale both the progress variable and resulting densityMassFractions
-     * @param densityProgressVariable
-     * @param predictedSourceEnergy , if null, wont' be set
-     * @param progressVariableSource , if null, won't be set
+     * @param densityProgressVariables
+     * @param densityEnergySource , if null, wont' be set
+     * @param densityProgressVariableSource , if null, won't be set
      * @param densityMassFractions , if null, won't be set
      */
-    void ChemTabModelComputeFunction(PetscReal density, const PetscReal densityProgressVariable[], PetscReal* predictedSourceEnergy, PetscReal* progressVariableSource,
+    void ChemTabModelComputeFunction(PetscReal density, const PetscReal densityProgressVariables[], PetscReal* densityEnergySource, PetscReal* densityProgressVariableSource,
                                      PetscReal* densityMassFractions) const;
+    void ChemTabModelComputeFunction(const PetscReal density[], const PetscReal* const* const densityProgressVariables, PetscReal** densityEnergySource, PetscReal** densityProgressVariableSource,
+                                     PetscReal** densityMassFractions, size_t batch_size) const;
+    // Batched Version of above
```

Comment:
Could you change this comment to work with doxygen for the method signature? Is this method used anywhere or tested?